docs/rules/no-constant-reassign.md

```diff
@@ -0,0 +1,54 @@
+# Disallow Reassignment of Constants (no-constant-reassign)
+
+Variables that are declared as constants but are later reassigned, are no longer constant, and should be declared with let or var.
```

Comment:
This section needs to show an example and mention `const`. The word "constant" is too ambiguous to stand alone.